### ISSUE TITLE
使用nss命令抓取firefox数据时如何写pcap文件？

### ISSUE THREAD

**Issue by LLIIUUKE** [gojue/ecapture#651]
nss命令选项写pcap从技术上可以实现吗？或者有没有其他方式可以实现抓取浏览器数据并写pcap文件的？

**Comment by cfc4n** [comment]
Of course, you can capture the traffic of the nss class library and save it as a pcapng file. This requires some R&D work. I need to capture the TLS communication key of the NSS class library. Can you send a PR?

**Comment by cfc4n** [comment]
hello ?
